### PR TITLE
bug fixes

### PR DESCRIPTION
**Pull request by T-Dourgarian**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,7 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
-
+/server/node_modules
```
Comment (T-Dourgarian) [inline .gitignore:7]: for future nestjs pushes

File: src/GameSlice.js [hunk 2]
```diff
@@ -74,6 +71,14 @@ export const GameSlice = createSlice({
       state.currentSeconds += 1;
     },
     getDailyKryptoSuccess: (state, action) => {
+      if (state.kryptoId !== action.payload.id) {
```
Comment (T-Dourgarian) [inline src/GameSlice.js:74]: this checks if it is a new day with new krypto data, and will reset relevant data. Was not being reset before

File: src/util/Formatter.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ export class Formatter {
   static getFormattedAverageTime = (seconds) => {
     const minutes = Math.floor(seconds / 60);
     const remainingSeconds = seconds % 60;
-    return seconds ? `${minutes}:${remainingSeconds}` : '0:00';
+    return seconds ? `${minutes}:${remainingSeconds < 10 ? '0' : ''}${remainingSeconds}` : '0:00';
```
Comment (wahlmichael) [inline src/util/Formatter.js:11]: :)

File: src/GameSlice.js [hunk 4]
```diff
@@ -74,6 +71,14 @@ export const GameSlice = createSlice({
...
+        state.currentSeconds = 0;
+        state.playedToday = false;
```
Comment (wahlmichael) [inline src/GameSlice.js:76]: This will show the how to play menu every day for the user. Is that what you want?

Comment (T-Dourgarian) [reply src/GameSlice.js:76]: For now, yah

File: src/GameSlice.js [hunk 5]
```diff
@@ -74,6 +71,14 @@ export const GameSlice = createSlice({
...
+        if (state.validSolutions.length === 0) {
+          state.solveStreak = 0;
```
Comment (wahlmichael) [inline src/GameSlice.js:78]: :/

**Review by wahlmichael — APPROVED** [review]

**Pull request merged**